src/generator/www/js/main.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use the function form of "use strict".](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l1)